### PR TITLE
Update dependencies, rename module again and fix some links

### PR DESCRIPTION
**Pull request by invidian**
Signed-off-by: Mateusz Gozdek <mateusz@kinvolk.io>

Locally run tests are passing, if this is something we can trust. Also basic operations with `ct` binary works fine as well.

### CODE REVIEW

**Review by krnowak — COMMENTED** [review]
I'm wondering if we should also move our dependency on ignition from `github.com/coreos/ignition` to our fork at `github.com/kinvolk/ignition`.

**Comment by invidian** [comment]
>I'm wondering if we should also move our dependency on ignition from github.com/coreos/ignition to our fork at github.com/kinvolk/ignition.

I think upstream Ignition is still maintained and we should rather update Flatcar to use latest version of it?

**Review by krnowak — APPROVED** [review]

**Comment by krnowak** [comment]
> > I'm wondering if we should also move our dependency on ignition from github.com/coreos/ignition to our fork at github.com/kinvolk/ignition.
> 
> I think upstream Ignition is still maintained and we should rather update Flatcar to use latest version of it?

True, I think that's planned. Not sure about the timeframe, though. :)

Anyway, let's get this in.

**Review by vbatts — APPROVED** [review]
seems fine

**Comment by invidian** [comment]
Actually, before we do that, I think we should pull all upstream changes first.

**Comment by invidian** [comment]
>Actually, before we do that, I think we should pull all upstream changes first.

Ah, this is already done in #6 :facepalm: I also pushed remaining tags from old upstream to avoid confusion.

**Comment by invidian** [comment]
Updated dependencies again, let's get this in.

File: go.mod [hunk 1]
```diff
@@ -1,20 +1,16 @@
-module github.com/flatcar-linux/container-linux-config-transpiler
+module github.com/kinvolk/container-linux-config-transpiler
 
-go 1.15
+go 1.16
 
 require (
-	github.com/ajeddeloh/go-json v0.0.0-20160803184958-73d058cf8437 // indirect
+	github.com/ajeddeloh/go-json v0.0.0-20200220154158-5ae607161559 // indirect
```
Comment (TechplexEngineer) [inline go.mod:6]: github.com/ajeddeloh/go-json redirects to https://github.com/coreos/go-json
Might be worth updating the dependency

File: go.mod [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
 	github.com/ajeddeloh/yaml v0.0.0-20170912190910-6b94386aeefd
```
Comment (TechplexEngineer) [inline go.mod:7]: This repo doesn't look maintained. Wonder if using a maintained fork such as https://github.com/go-yaml/yaml would make sense

File: go.mod (hunk 1 above)
Comment (invidian) [reply go.mod:6]: Thanks for suggesting @techplexEngineer. I think this project requires a lot more of maintenance tasks, which needs to be addressed, so perhaps we can address this as well at later stage.

**Pull request merged**